modules/services/window-managers/bspwm/options.nix

```diff
@@ -177,6 +177,14 @@ in {
       '';
     };
 
+    extraConfigEarly = mkOption {
+      type = types.lines;
+      default = "";
+      description = ''
+        Like extraConfig, except commands are run at the start of the config
+                file.'';
```

Comment:
Looks odd.

Reply:
Fixed in b7a6ebcf16cecff1bf9a2bd4fbeb8d6f422e04fb.